app/models/EmailForm.scala

```diff
@@ -0,0 +1,18 @@
+package models
+
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.libs.json.OFormat
+
+
+case class EmailForm(email: String, subject: String, message: String)
+
+object EmailForm {
+  val email: Form[EmailForm] = Form(
+    mapping(
+      "email" -> nonEmptyText,
```

Comment:
```suggestion
      "email" -> email,
```

Reply:
In the brief validating the email is one of the requirements

Reply:
Done